### DISCUSSION TITLE
首页可切换主题样式配色的按钮不见了

### DISCUSSION THREAD

**Discussion by levarsi** [halo-dev/halo#7140] [General]
首页可切换主题样式配色的按钮不见了

![error](https://github.com/user-attachments/assets/093f9dac-553f-4dfc-a4f3-1391c093dae9)

![huanjing](https://github.com/user-attachments/assets/9b170f1b-b36e-4a45-8bee-75c7cfce09e2)

**Comment by levarsi** [comment]
![error1](https://github.com/user-attachments/assets/f3f07a69-3d1a-4cd0-9388-3ac70438cd21)

**Comment by ruibaby** [comment]
https://github.com/halo-dev/halo/issues/6864
